website/docs/docs/develop/getting-started/learning-more.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Learning more"
+title: "Learn more about dbt"
```

Comment:
i'm unsure if this page is necessary as i click around the left sidebar. the page is mostly a topic list  which the left sidebar already provides so it seems redundant, low-value. consider not including it.